pkg/virt-controller/watch/workload-updater/workload-updater.go

```diff
@@ -31,6 +30,7 @@ import (
 	"kubevirt.io/client-go/log"
 
 	"kubevirt.io/kubevirt/pkg/controller"
+	metrics "kubevirt.io/kubevirt/pkg/monitoring/metrics/virt-controller"
```

Comment:
nit: I think we can remove the empty lines (line 25, 27, 31) since all is kubevirt.io

Reply:
nice catch

although `"kubevirt.io/client-go/...` imports are usually separated, but yes, a few others out of place

anyway I think that is not very related to this PR, maybe could be done in a separate one 
by you maybe :D